cartridge-cli.lua

```diff
@@ -968,6 +968,21 @@ local function tarantool_is_enterprise()
     return fio.path.exists(tnt_version)
 end
 
+local function get_sdk_version()
+    assert(tarantool_is_enterprise())
+
+    local version_filepath = fio.pathjoin(get_tarantool_dir(), 'VERSION')
+    local version_filepath_content = read_file(version_filepath)
+
+    local sdk_version = string.match(version_filepath_content, 'TARANTOOL_SDK=(%S+)\n')
+    if sdk_version == nil then
+        die('Failed to get SDK version from %s file', version_filepath)
```

Comment:
Why not `nil, err`? We should respect error handling convention

Reply:
This function is called outside of pack handler, this is the part of setting up `app_state`.